index.html

```diff
@@ -0,0 +1,222 @@
+          <a class="nav-link" href="#What_you_should_already_know">What you should already know</a>
+          <a class="nav-link" href="#JavaScript_and_Java">JavaScript and java</a>
+          <a class="nav-link" href="#Hello_world">Hello World</a>
+          <a class="nav-link" href="#Variables">Variables</a>
+          <a class="nav-link" href="#Declaring_variables">Declaring variables</a>
+          <a class="nav-link" href="#Variable_scope">Variable Scope</a>
+          <a class="nav-link" href="#Global_variables">Global variables</a>
+          <a class="nav-link" href="#Constants">Constants</a>
+          <a class="nav-link" href="#Data_types">Data types</a>
+          <a class="nav-link" href="#if...else_statement">if...else statement</a>
+          <a class="nav-link" href="#while_statement">While statement</a>
+          <a class="nav-link" href="#Function_declarations">Function declarations</a>
+          <a class="nav-link" href="#Reference">Reference</a>
+    </nav>
+  </div>
+
+  <!--Main Page-->
+  <main id="main-doc">
+  <section class="main-section" id="Introduction">
+    <header>Introduction</header>
+      <p>JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language. Inside a host environment (for example, a web browser), JavaScript can be connected to the objects of its environment to provide programmatic control over them.</p>
+      <p>JavaScript contains a standard library of objects, such as Array, Date, and Math, and a core set of language elements such as operators, control structures, and statements. Core JavaScript can be extended for a variety of purposes by supplementing it with additional objects; for example:</p>
+      <ul>
+      <li>Client-side JavaScript extends the core language by supplying objects to control a browser and its Document Object Model (DOM). For example, client-side extensions allow an application to place elements on an HTML form and respond to user events such as mouse clicks, form input, and page navigation.</li>
+      <li>Server-side JavaScript extends the core language by supplying objects relevant to running JavaScript on a server. For example, server-side extensions allow an application to communicate with a database, provide continuity of information from one invocation to another of the application, or perform file manipulations on a server.</li>
+    </ul>
+ </section>
+
+
+ <section class="main-section" id="What_you_should_already_know">
+  <header>What you should already know</header>
+    <p>This guide assumes you have the following basic background:</p>
+    <ul>
+
+      <li>A general understanding of the Internet and the World Wide Web (WWW).</li>
+      <li>Good working knowledge of HyperText Markup Language (HTML).</li>
+      <li>Some programming experience. If you are new to programming, try one of the tutorials linked on the main page about JavaScript.</li>
+    </ul>
+</section>
+
+
```

Comment:
Remove empty spaces

Reply:
- [ ] Kindly always include an explanation, or a "why", for each requested change, this will make your comments more useful and clear. eg why having the empty lines in your code base a bad thing